src/cli.js

```diff
@@ -17,16 +17,16 @@ didYouMean.threshold = 0.6;
 import initializeProject from './commands/basic/init';
 import generateFile from './commands/basic/generate';
 import asyncRender from './commands/basic/codesplit';
-import addPlugins from './commands/basic/add';
+import addDeps from './commands/basic/add';
 import setupProject from './commands/serve/setup';
 import dockerize from './commands/basic/docker';
 import deployConfig from './commands/deploy/deploy';
 import pkg from '../package';
 
 updateNotifier({ pkg }).notify();
 
-const suggestCommands = cmd => {
-  const availableCommands = program.commands.map(c => c._name);
+const suggestCommands = (cmd) => {
```

Comment:
- do we need the bracket i guess if it's just a single para  we can go with `cmd => {}`

Reply:
It is in accordance with the `prettier` config.